resources/web/style/toc.pcss

```diff
@@ -0,0 +1,86 @@
+#guide {
+  .toc {
+    ul {
+      margin: 0 0 1em 0;
+      padding: 0;
+      border: 1px solid #ddd;
+      ul {
+        /* Hide all sub-toc elements by default. See rules with `.show` that
```

Comment:
I feel like being able to add comments here and know they won't bloat the download is a nice win.